### ISSUE TITLE
import-alias-naming incorrect?

### ISSUE THREAD

**Issue by marcelloh** [mgechev/revive#919]
**Describe the bug**
https://go.dev/blog/package-names
On this page should be something mentioned about alias naming, but I can't find it.
Also:
when I try this:
^[a-z][\\w]{0,}$
it still complains about libTracer not matching it.
import-alias-naming: import name (libTracer) must match the regular expression: ^[a-z][\w]{0

**To Reproduce**
I have it running from golangci-lint

```yaml
      # https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#import-alias-naming
      - name: import-alias-naming
        arguments: "^[a-z][\\w]{0,}$"
```

**Expected behavior**
I expect a working solution, where revive doesn't complain about my imports.

**Logs**
-

**Desktop (please complete the following information):**
 - OS: MacOs Sonoma 14.0
 - Version of Go: go version go1.21.3 darwin/arm64

**Additional context**
-

**Comment by marcelloh** [comment]
found out that this works, which is good enough for me:
"^[a-z][a-zA-Z0-9]*$"

**Comment by denisvmedia** [comment]
@marcelloh the page you are referring indeed only speaks about the package names, but is there any good reason for the aliases to not follow the idiomatic rules of the package names? In the end alias is an alias of the package name, in other words it is a package name as well.

**Comment by marcelloh** [comment]
As it works for me now with a proper regex, I feel that it doesn't matter.
(and not so much fan of _)
